### DISCUSSION TITLE
Cloning item between organization doesn't work

### DISCUSSION THREAD

**Discussion by luciolebrillante** [dani-garcia/vaultwarden#3016] [Q&A]
### Subject of the issue
When I want to clone an item from an organization to another organization, I can't select a collection from the organization I wanted to move the object.

### Deployment environment
    Vaultwarden version: v1.26.0
    Web-vault version: v2022.10.0
    Running within Docker: false (Base: Debian)
    Environment settings overridden: true
    Uses a reverse proxy: true
    IP Header check: true (X-Real-IP)
    Internet access: true
    Internet access via a proxy: false
    DNS Check: true
    Time Check: true
    Domain Configuration Check: true
    HTTPS Check: true
    Database type: PostgreSQL
    Database version: PostgreSQL 14.5 (Debian 14.5-2.pgdg110+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit
    Clients used:
    Reverse proxy and version:
    Other relevant information:

* Clients used: Web

* Reverse proxy and version: Nginx

* MySQL/MariaDB or PostgreSQL version: Postgresql

* Other relevant details:

### Steps to reproduce
Go in an organisation you own.
Click on the ...
Select Clone
Who own this item : choose another organization
![image](https://user-images.githubusercontent.com/50231120/208710455-7633bf6b-6d5e-4f88-a437-c829d52a5066.png)

### Expected behaviour
When a user selects another organization, all the collections which he has right to write should be displayed.

### Actual behaviour
When a user selects another organization, no collections are displayed.

**Comment by BlackDex** [comment]
This seems to be expected behavior: https://github.com/bitwarden/clients/issues/2704

**Reply by jmechnich** [reply]
The referenced issue is about cloning using the browser extension. There I cannot even seem to find a cloning option.
However, coming back to the issue above, this looks like a bug to me.

A possible but a bit clumsy workaround is:

- clone item to private vault
- move item to final destination in a different organization

Originally I wanted to move an item straight from one organization to another, which seems to be possible from the GUI but throws an error on execution (see #3771).

**Reply by BlackDex** [reply]
You can't move items between organizations.
You can only move personal items to an organization.

You probably try to selectbox/multi-select workflow via your personal vault overview to do this.
Which kinda seems to indicate that it would work, but it is not.

You first need to clone the item back to your personal vault, and then you can move that item to a different organization.

**Comment by quadcom** [comment]
Ran into this today. 

I feel that as the server admin, they should be able to move items from org to org, at least in the webvault. 

The current workaround is rather sloppy and slow. It's way too easy to lose a cloned item in the personal vault and have to hunt it down to move it to the other org.

Please reconsider this for a fix.

Thx for such an awesome prod!

**Reply by BlackDex** [reply]
I can be short on this. This project it's self can't fix this on the server side. The clients just do not allow this. The ciphers need to be re-encrypted with a different key, which is not something the server side is able to do.

**Reply by quadcom** [reply]
Sorry, I thought I was posting this on the bitwarden git.

**Reply by quadcom** [reply]
As I anticipated, they threw it back onto you to fix (which I understand, you can't).

https://github.com/bitwarden/server/issues/3939

**Reply by BlackDex** [reply]
Best is to test this via the cloud environment and use that as the use case. It is the only way which will benefit all users.

Personally, i just don't have the time to also learn Typescript in such detail so that I can contribute to Bitwarden. If i could i would.

**Reply by Gerardv514** [reply]
That is because you mentioned you’re having the issue on Vaultwarden and neither of the two are associated with the other. 

You need to indicate that this is also occurring on Bitwarden. I believe there is also a discussion, maybe feature request, on their community forum already. Find that and reference the bitwarden community post which indicates what you described is happening on Bitwarden.
